Eric wang/usage attribution docs (#21885)
DataDog/documentation: content/en/account_management/billing/usage_attribution.md

Usage Attribution is an advanced feature included in the Enterprise plan. For all other plans, contact your account representative or <a href="mailto:success@datadoghq.com">success@datadoghq.com</a> to request this feature.
</div>

Administrators or users with the Usage Read permission can access the Usage Attribution tab from the Plan & Usage section in Datadog. The Usage Attribution page provides the following information and functionality:

- Lists the existing tag keys that usage is being broken down by and provides the ability to change and add new ones (up to three tag keys).
- Summarizes usage at the end of each month and visualizes usage over time broken out by tags.
- Generates month-to-date and hourly CSV files.

This feature does not support product usage that cannot be tagged during instrumentation. For example, Incident Management Users, CI Pipeline and Test Users, Parallel Testing Slots, and Audit Trail. 

## Getting started

> To start receiving daily data, an administrator needs to create a select tags for the report.

Reviewer: ```suggestion
To start receiving daily data, an administrator needs to choose tags for the report.
```
Is my edit correct?

- Monthly reports reflect the latest set of tags. If tags are changed in the middle of a month, partial month reports are created for each reporting period.

## Total usage

### Monthly usage attribution

Monthly reports are updated daily and provide a month-to-date aggregation of usage data.

{{< img src="account_management/billing/usage_attribution/Usage-Attribution-Monthly-Facets.png" alt="Applied tags in Datadog" style="width:100%;" >}}

- Data for specific products, tags, and organizations can be selected using the facet selector.
- Data can be grouped and ungrouped by the tag keys selected.
- Value and Percentage options are available for table display. 
- Data shown on the table can be edited to include select products. 
- If multi-org is enabled, usage is summarized across all Datadog organizations at the parent account.
- Previous months' reports are accessible through the time selector.
- Reports are downloadable in CSV format. These CSV reports include both usage numbers and percentages, allowing for simplified allocations and chargebacks. Percentages are calculated on a per-organization basis.

> Monthly data can also be pulled via API. For more information, see the [API endpoint documentation][1].

Reviewer: ```suggestion
Monthly data can also be pulled using the API. For more information, see the [API endpoint documentation][1].
```

### Hourly usage attribution

> Hourly data can be pulled via API. For more information, see the [API endpoint documentation][2].

Reviewer: ```suggestion
Hourly data can be pulled using the API. For more information, see the [API endpoint documentation][2].
```

## Tracking usage

A timeseries of Usage Attribution data can be viewed by clicking on "Track Usage"
- Data for specific products, organization, or tag keys can be selected using the facet selector.
- Data can be graphed for a day, week, or month by using the time selector above the graphs.

{{< img src="account_management/billing/usage_attribution/Usage-Attribution-Hourly-Facets.png" alt="Infra Hosts graphs separated by tags" style="width:100%;" >}}

> ## Cost Attribution

Reviewer: ```suggestion
## Cost attribution
```

For direct billing customers, month-end cost attribution reports are generated at the end of each billing cycle to enable monthly chargeback and cost allocation processes. 
- Cost data for the preceding month is available no later than the 19th of the current month.
- Cost attribution data is not currently available in GovCloud datacenters

> - Monthly Cost Attribution data is [available via API][4]

Reviewer: ```suggestion
- Monthly Cost Attribution data is [available with the API][4]
```
